tests/acceptance/features/user/super_user/pending_usertopoflist.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Pending new user should be display on top of user list page
+
+    @public-user @user @add-user
```

Comment:
Should include a unique identifier like "@7620-0007". You can use a variant of the time and date.